xinference/model/llm/llm_family.json

```diff
@@ -617,5 +617,67 @@
       }
     ],
     "prompt_style": null
+  },
+  {
+    "version": 1,
+    "model_name": "falcon-instruct",
+    "model_lang": [
+      "en"
+    ],
+    "model_ability": [
+      "embed",
+      "chat"
+    ],
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_id": "tiiuae/falcon-7b-instruct"
+      }
+    ],
+    "prompt_style": {
+      "style_name": "FALCON",
+      "roles": [
+        "User",
+        "Assistant"
+      ],
+      "intra_message_sep": "\n"
+    }
+  },
+  {
+    "version": 1,
+    "model_name": "falcon-instruct",
+    "model_lang": [
+      "en"
+    ],
+    "model_ability": [
+      "embed",
+      "chat"
+    ],
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 40,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_id": "tiiuae/falcon-40b-instruct"
+      }
+    ],
+    "prompt_style": {
+      "style_name": "FALCON",
+      "roles": [
+        "User",
+        "Assistant"
+      ],
+      "intra_message_sep": "\n"
+    }
```

Comment:
- prompt_style
```
"prompt_style": {
      "style_name": "FALCON",
      "system_prompt": "",
      "roles": [
        "User",
        "Assistant"
      ],
      "intra_message_sep": "\n",
      "inter_message_sep": "<|endoftext|>",
      "stop": ["\nUser"],
      "stop_token_ids": [
        0,
        1,
        2,
        3,
        4,
        5,
        6,
        7,
        8,
        9,
        10,
        11
      ]
    }
```
- `FALCON` style_name:
```
        elif prompt_style.style_name == "FALCON":
            ret = prompt_style.system_prompt
            for message in chat_history:
                role = message["role"]
                content = message["content"]
                if content:
                    ret += (
                            role
                            + ": "
                            + content.replace("\r\n", "\n").replace("\n\n", "\n")
                    )
                    ret += "\n\n"
                else:
                    ret += role + ":"
            return ret
```
- add unittest for `FALCON` style_name